packages/k2-alpine/src/components/Avatar/HexagonImageView.tsx

```diff
@@ -20,26 +20,35 @@ import {
   lightModeColors
 } from '../../theme/tokens/colors'
 import { useTheme } from '../../hooks'
+import { Icons } from '../../theme/tokens/Icons'
+import { View } from '../Primitives'
+
+type TestnetIconSize = 'small' | 'large'
 
 export const HexagonImageView = ({
   source,
   height,
   backgroundColor,
   isSelected,
-  hasLoading = false
+  hasLoading = false,
+  isDeveloperModeEnabled = false,
+  testnetIconSize = 'small'
```

Comment:
nit: I feel like these props are not the right props for this generic component. maybe creating another component or a wrapper around HexagonImageView